### PR TITLE
Fix(layout.html): Add logo w nav to home

### PR DESCRIPTION
**Pull request by Hajar1**
## Description of changes
added logo in the middle of nav back that gives the user the ability to go back to the home screen

## Screenshots (If Applicable)
Screen Shot 2019-03-19 at 5 48 12 PM

## Checklist
- [ ] I have unit tested my code.
- [ ] I have integration tested my code.
- [ ] All new and existing tests passed.
- [ ] I have [updated the diagrams](https://github.com/amazin-team/Amazin-online-bookstore/wiki/Updating-the-diagrams) ModelsUMLClassDiagram and EntityRelationshipDiagram (if necessary).

### CODE REVIEW

**Review by LauraMachado21 — COMMENTED** [review]
You could move the navbar to a fragment, like your previous PR. But, it is not required and is completey up to you.

Also, in my opinion, the logo should go on the very left of the sidebar (to the left of the search). @abungay what do you think?

**Comment by Hajar1** [comment]
@LauraMachado21 

> You could move the navbar to a fragment, like your previous PR. But, it is not required and is completely up to you.
>  That can be done in another pull request
> Also, in my opinion, the logo should go on the very left of the sidebar (to the left of the search). @abungay what do you think?
> You could but you'd end up with lots of empty white in the middle

**Comment by abungay** [comment]
Here's another possible styling approach we could take to this:

- leave in the AMAZIN text logo title on the main page
- have the book icon on the left side of the navbar next to the search bar with a small vertical divider separating the two

sample styling prototype:
possiblenavlayout

@LauraMachado21  @Hajar1

**Comment by abungay** [comment]
Most websites seem to follow the convention of having the clickable logo on the far left of the navigation bar next to the search bar and having the user login at the top right.

e.g. GitHub
githubnav

**Comment by Hajar1** [comment]
> Here's another possible styling approach we could take to this:
> 
> * leave in the AMAZIN text logo title on the main page
> * have the book icon on the left side of the navbar next to the search bar with a small vertical divider separating the two
> 
> sample styling prototype:
> possiblenavlayout
> 
> @LauraMachado21 @Hajar1

I'd move the logo to the left buy the AMAZIN at the middle is not useful anymore it should be added in a different pr to the login page

**Comment by abungay** [comment]
Amazon.ca takes the same approach as mentioned above with their logo on the far left, search bar next to it and user login at the right side. However, since search is a very common feature for Amazon.ca, they make the search bar resize to take up the whole center of the screen. @Hajar1 this could prevent the excess whitespace in the middle of the screen if we do it like Amazon.ca

e.g.
amazonnav

**Comment by abungay** [comment]
I also just noticed that one of the features of amazon's large search bar in their navigation is the ability to select a category to search through. That may be a nice future feature for us to add.

@zaidoon1

**Comment by LauraMachado21** [comment]
> > Here's another possible styling approach we could take to this:
> > 
> > * leave in the AMAZIN text logo title on the main page
> > * have the book icon on the left side of the navbar next to the search bar with a small vertical divider separating the two
> > 
> > sample styling prototype:
> > possiblenavlayout
> > @LauraMachado21 @Hajar1
> 
> I'd move the logo to the left buy the AMAZIN at the middle is not useful anymore it should be added in a different pr to the login page

Where would the logo direct to from the login page? We don't have a "homepage"

**Comment by Hajar1** [comment]
Screen Shot 2019-03-19 at 6 45 32 PM

**Comment by Hajar1** [comment]
> > > Here's another possible styling approach we could take to this:
> > > 
> > > * leave in the AMAZIN text logo title on the main page
> > > * have the book icon on the left side of the navbar next to the search bar with a small vertical divider separating the two
> > > 
> > > sample styling prototype:
> > > possiblenavlayout
> > > @LauraMachado21 @Hajar1
> > 
> > 
> > I'd move the logo to the left buy the AMAZIN at the middle is not useful anymore it should be added in a different pr to the login page
> 
> Where would the logo direct to from the login page? We don't have a "homepage"

There is no navbar when a user trying login or signup currently

**Comment by abungay** [comment]
One small comment on the latest with the logo on the left side of the navbar from @LauraMachado21: the book logo resolution isn't the best so maybe we should make it a bit smaller.

File: src/main/resources/public/css/stylesheet.css [hunk 1]
```diff
@@ -9,9 +9,14 @@ input, textarea { display: table-cell; }
     justify-content: center;
 }
 
-#logo {
-    font-size: 90px;
-    margin: 0;
+#icon {
+    display: block;
+    margin-left: 2.5%;
```
Comment (LauraMachado21) [inline src/main/resources/public/css/stylesheet.css]: The logo will be floating towards the middle as you increase the size of the screen. Please use px or em instead. (I suggest 1em)

**Review by LauraMachado21 — DISMISSED** [review]

File: src/main/resources/public/css/stylesheet.css (hunk 1 above)
Comment (abungay) [reply src/main/resources/public/css/stylesheet.css]: fixed, changed to 1em margins

**Review by abungay — APPROVED** [review]

**Pull request merged**